docs/stencil-docs/reference-docs/handlebars-helpers-reference.md

```diff
@@ -14,6 +14,7 @@ The following table contains BigCommerce's open source [Handlebars helpers](http
 | [pluck](#pluck) | array | Uses search key to get values from collections. |
 | [cdn](#cdn) | assets | A URL transformer for content delivery networks. |
 | [money](#money) | currency | Formats number length, thousands delimiter, and decimal delimiter. |
+| [moment](#moment) | date | Use [moment](https://momentjs.com/) as a helper. |

```

Comment:
```suggestion
| [moment](#moment) | date | Use [momentjs](https://momentjs.com/) as a helper. |

```
Everywhere you're referring to the library, use `momentjs`. When you're referring to the date objects, `moment` is correct.